src/server/enrichment-map/enrichment/enrichment.js

```diff
@@ -11,17 +11,17 @@ const _ = require('lodash');
 const qs = require('query-string');
 
 
-// remove #WARNING, #INFO, and the first line
 const parseGProfilerResponse = (gProfilerResponse) => {
-  // remove the second line
-  const lines = gProfilerResponse.split('\n');
-  lines.slice(0, 1);
-  const str1 = gProfilerResponse.split('\n').slice(0, 1).join("\n");
-  let str2 = gProfilerResponse.split('\n').slice(2).join("\n"); // concatenate at last
-  // remove lines starting with #
-  const str3 = str2.replace(/^#.*$/mg, "");
-  const str4 = (str1 + '\n').concat(str3);
-  return str4;
+  let lines = _.map(gProfilerResponse.split('\n'), line => {
+    if (line.substring(0, 1) === '#') {
```

Comment:
Are you checking if each line starts with '#'?

Reply:
Yes